### PR TITLE
Update Inertia middleware example

### PR DESCRIPTION
**Pull request by afgallo**
This PR updates the Inertia middleware example in the AdonisJS documentation: https://docs.adonisjs.com/guides/views-and-templates/inertia#share-from-a-middleware

### CODE REVIEW

**Comment by RomainLanz** [comment]
Hey @afgallo! 👋🏻 

What's wrong with the current middleware?

**Comment by afgallo** [comment]
> Hey @afgallo! 👋🏻
> 
> What's wrong with the current middleware?

Hey @RomainLanz,

I guess there is nothing wrong with the middleware itself, the documentation doesn't reflect the method signature which is what I've updated in the docs with this PR 👍🏻 

image

image

File: content/docs/views-and-templates/inertia.md [hunk 1]
```diff
@@ -518,9 +518,11 @@ import type { HttpContext } from '@adonisjs/core/http'
...
 
 export default class MyMiddleware {
-  public async handle({ inertia }: HttpContext, next: NextFn) {
-    inertia.share('appName', 'My App')
-    inertia.share('user', (ctx) => ctx.auth?.user)
+  public async handle({ inertia, auth }: HttpContext, next: NextFn) {
+    inertia.share({ appName: 'My App' })
+    await auth.check()
+    const user = auth.user
+    inertia.share({ user: { id: user?.id, email: user?.email } })
```
Comment (RomainLanz) [inline content/docs/views-and-templates/inertia.md:525]:
```suggestion
  public async handle({ inertia, auth }: HttpContext, next: NextFn) {
    inertia.share({
      appName: 'My App',
      user: (ctx) => ctx.auth?.user
    })
```

**Comment by RomainLanz** [comment]
Oh yeah, the signature is wrong. It has to be an object.
I have changed a bit the example to be less verbose and more accurate of the previous one.

**Comment by RomainLanz** [comment]
Thanks!

File: content/docs/views-and-templates/inertia.md [hunk 2]
```diff
@@ -518,9 +518,11 @@ import type { HttpContext } from '@adonisjs/core/http'
 import type { NextFn } from '@adonisjs/core/types/http'
 
 export default class MyMiddleware {
-  public async handle({ inertia }: HttpContext, next: NextFn) {
-    inertia.share('appName', 'My App')
-    inertia.share('user', (ctx) => ctx.auth?.user)
+  public async handle({ inertia, auth }: HttpContext, next: NextFn) {
```
Comment (thetutlage) [inline content/docs/views-and-templates/inertia.md:521]: We don't use `public` modifiers anywhere. So it will be nice to keep docs uniformed :)

Comment (thetutlage) [reply content/docs/views-and-templates/inertia.md:521]: I did that already in this commit :) https://github.com/adonisjs/v6-docs/commit/33a9cfc153b0514fe16d4ae19e1ba7ef742a1746

**Pull request merged**